models/lobby_test.go

```diff
@@ -158,6 +158,20 @@ func TestReadyPlayer(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+func TestIsEveryoneReady(t *testing.T) {
```

Comment:
you should also test a case where everyone is ready. Use a loop to create 12 players, ready them up and then assert IsEveryoneReady==true